.github/prompts/advanced-best-practices-dp.prompt.md

```diff
@@ -0,0 +1,22 @@
+I've recently converted my swagger specification to typespec, can you go through the files of my project and make sure they're following the best practices when doing a data-plane migration?
```

Comment:
how is this different than the guidance for converting a management plane spec?

Reply:
This file was actually added by mistake, removed now. In the future we can have other prompts that tell the agent to do more clean up or even larger refactoring.